src/app/(auth)/onboarding/layout.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+
+import BigButton from '@/components/BigButton';
+import ProgressBar from '@/components/ProgressBar';
+import { ReactNode } from 'react';
+
+export default function layout({ children }: { children: ReactNode }) {
+  // track progress with onboarding context
+  return (
+    <>
+      <ProgressBar
+        steps={new Set(['Basic Info', 'Languages', 'Legal Experience', 'Done'])}
+        progress={2}
+      />
+      {children}
+      <BigButton>Continue</BigButton>
```

Comment:
Can we make the text on this button white?

Also, curious how putting the button in the `layout.tsx` file would affect routing?

Reply:
I believe the text color is a part of the Big Button sprint that is still in the backlog. The button placed here is because, on the Figma, there's a button at the bottom of all the onboarding pages that all said "Continue" (except for the last one 😔). We could remove it, up for discussion. Routing-wise, I'm thinking of handling the complexity in the onboarding `layout.tsx` and communicating with the onboarding pages using the onboarding context.